_includes/m2/solution_validation.html

```diff
@@ -8,10 +8,46 @@ <h2 class="section-heading text-uppercase">{{ site.data.m2.solution_validation.t
       </div>
   
       <div class="row">
-        <div class="col col-lg-12 text-center">
-          <p>Customers are nice. We want many of them. Also don't write text here. Put text and picture paths in _data/m2.yml under the specific section :)</p>
+        <div class="col col-lg-12 text-justify">
+            <p>{{ site.data.m2.solution_validation.introtext }}</p>
+
+            {% for l in site.data.m2.solution_validation.interview %}
+              <div class="row">
+                <div class="col col-lg-12 text-center">
+                    <audio controls>
+                        <source src="assets/audio/solution_validation/{{ l.audio }}"
```

Comment:
Let's link the audio files from google drive isntead of uploading them to github.
[Check point 3 in this SO answer for the format of the URL.](https://stackoverflow.com/a/54865891/1290494)